.github/workflows/ci.yml

```
@@ -65,11 +65,10 @@ jobs:
             echo "BUF_BASE_BRANCH=$GITHUB_BASE_REF" >> $GITHUB_ENV
           fi
 
-      - uses: bufbuild/buf-setup-action@v1
+      - name: Proto buf checks
```

"Protobuf" is one word:

```suggestion
      - name: Protobuf checks
```

---

Lol. I was going for the precision pun as the tool is called 'buf'. Obviously fell flat.